### ISSUE TITLE
A question about fine-tuning

### ISSUE THREAD

**Issue by Mr00guo** [zhanglichao/end2end_rgbt_tracking#7]
Excuse me，I have one question about the fine-tuning.  You mentioned "We keep the default learning rates for each component as in the DiMP model and then decrease them by collaboratively multiplying a small gain learning rate, i.e. 0.001 when fine-tuning" during the training phase of mfDiMP. How to perform fine-tuning operation? Did you use the converted GOT-10k to train according to the default value of dimp, and then multiply the learning rate by 0.001 to train again? Or at the beginning of training using the converted GOT-10k, multiply the default learning rate by 0.001? Would you please answer me the  question ? Thanks!

**Comment by missyoudaisy** [comment]
I have the same question！

**Comment by missyoudaisy** [comment]
I see the code！ I found that maybe the author used files in '/end2end_rgbt_tracking/mfDiMP/ltr/train_settings/seq_tracking/sdlearn_300_onlytestloss_lr_causal_mg30_iou_nocf_res50_lfilt512_coco.py' to train the RGB modality tracker. And then use the pretrained model in epoch 40 to finetune.

**Comment by Mr00guo** [comment]
> I see the code！ I found that maybe the author used files in '/end2end_rgbt_tracking/mfDiMP/ltr/train_settings/seq_tracking/sdlearn_300_onlytestloss_lr_causal_mg30_iou_nocf_res50_lfilt512_coco.py' to train the RGB modality tracker. And then use the pretrained model in epoch 40 to finetune.

The first 40 epochs only trained RGB's feature extraction network and other components? How did the first 40 EPOchs deal with the infrared feature extraction network? May I have your contact information?

**Comment by missyoudaisy** [comment]
I see the code！ I found that maybe the author used files in '/end2end_rgbt_tracking/mfDiMP/ltr/train_settings/seq_tracking/sdlearn_300_onlytestloss_lr_causal_mg30_iou_nocf_res50_lfilt512_coco.py' to train the RGB modality tracker. And then use the pretrained model in epoch 40 to finetune.

**Comment by missyoudaisy** [comment]
> > I see the code！ I found that maybe the author used files in '/end2end_rgbt_tracking/mfDiMP/ltr/train_settings/seq_tracking/sdlearn_300_onlytestloss_lr_causal_mg30_iou_nocf_res50_lfilt512_coco.py' to train the RGB modality tracker. And then use the pretrained model in epoch 40 to finetune.
> 
> The first 40 epochs only trained RGB's feature extraction network and other components? How did the first 40 EPOchs deal with the infrared feature extraction network? May I have your contact information?

In my view, the author firstly used sdlearn_300_onlytestloss_lr_causal_mg30_iou_nocf_res50_lfilt512_coco.py to train for RGB tracker, and then use this RGB pretrained model in 40th epoch and other python files to finetune for RGBT trakcer. I only have wechat \qq\ email.

**Comment by Mr00guo** [comment]
> > > I see the code！ I found that maybe the author used files in '/end2end_rgbt_tracking/mfDiMP/ltr/train_settings/seq_tracking/sdlearn_300_onlytestloss_lr_causal_mg30_iou_nocf_res50_lfilt512_coco.py' to train the RGB modality tracker. And then use the pretrained model in epoch 40 to finetune.
> > 
> > 
> > The first 40 epochs only trained RGB's feature extraction network and other components? How did the first 40 EPOchs deal with the infrared feature extraction network? May I have your contact information?
> 
> In my view, the author firstly used sdlearn_300_onlytestloss_lr_causal_mg30_iou_nocf_res50_lfilt512_coco.py to train for RGB tracker, and then use this RGB pretrained model in 40th epoch and other python files to finetune for RGBT trakcer. I only have wechat \qq\ email.

thank you.My qq is 794770789,Please contact me if it is convenient for you
